server/stream.go

```diff
@@ -120,6 +120,9 @@ type StreamConfig struct {
 	// to the stream, improving throughput.
 	AllowAsyncFlush bool `json:"allow_async_flush"`
 
+	// Controls whether or not the stream manages its consumers instead of the metalayer.
+	ManagesConsumers bool `json:"managed_consumers,omitempty"`
```

Comment:
Let's use `ManagesConsumers` / `manages_consumers` throughout?
```suggestion
	ManagesConsumers bool `json:"manages_consumers,omitempty"`
```